en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -162,7 +169,7 @@ Let's design the REST API as follows:
 
     3. Click **Save**.
 
-6. To build the JSON payload to be sent as the response and then send the response, follow this procedure:
+8. To build the JSON payload to be sent as the response and then send the response, follow this procedure:
 
     1. To build the JSON payload with data of the total cases per million in the population, add a variable.
```

Comment:
Since we have just one step here, should we rephrase the step above in a way that we don't need to say ""follow this procedure"?
 WDYT?

Reply:
Re-arranged the bullet points here.